rosdep/python.yaml

```diff
@@ -7098,6 +7098,15 @@ qdarkstyle-pip:
     pip: [qdarkstyle]
   ubuntu:
     pip: [qdarkstyle]
+qtpy-pip:
+  debian:
```

Comment:
It looks like at least on Debian, there is a distribution package for this: https://packages.debian.org/buster/python3-qtpy (this may be the case on other Linux distributions as well; I didn't check).  Since we prefer the distribution package, please update this key to use the distribution package instead.

Reply:
@chikuta Friendly ping. Can you update the keys to use the distribution package where available? Please post links to the web listings for making reviewing easier (as described in [contributing.md](https://github.com/ros/rosdistro/blob/master/CONTRIBUTING.md#rosdep-rules-contributions)).

Reply:
@jacobperron Thank you for your ping. I overlooked your comments. I will fix it as soon as possible.

Reply:
- [ ] write qtpy as distribution package

Reply:
* python2
  * arch (**found**) : https://archlinux.org/packages/?sort=&q=python-qtpy&maintainer=&flagged=
  * debian (**found** on stretch, buster) :  https://archlinux.org/packages/?sort=&q=python-qtpy&maintainer=&flagged=
  * fedora (**found** on 32, 33) : https://src.fedoraproject.org/rpms/python-QtPy
  * gentoo (**not found** target python version are 3.7, 3.8, 3.9) :  https://packages.gentoo.org/packages/dev-python/QtPy
  * ubuntu (**found** on bionic) : https://packages.ubuntu.com/search?suite=all&section=all&arch=any&keywords=python-qtpy&searchon=names

* python3
  * arch (**not found**) :  https://archlinux.org/packages/?sort=&q=python3-qtpy&maintainer=&flagged=
  * debian (**not found**) :  https://archlinux.org/packages/?sort=&q=python3-qtpy&maintainer=&flagged=
  * fedora (**not found**) : https://src.fedoraproject.org/rpms/python3-QtPy
  * gentoo (**found** target python version are 3.x) :  https://packages.gentoo.org/packages/dev-python/QtPy
  * ubuntu (**found** on bionic, focal, groovy) : https://packages.ubuntu.com/search?suite=all&section=all&arch=any&keywords=python3-qtpy&searchon=names